tensorflow/dtensor/mlir/utils/collective_lowering.cc

```diff
@@ -1232,6 +1437,28 @@ struct DTensorAllScatterLowering
   }
 };
 
+struct DTensorAllToAllLowering
+    : public impl::DTensorAllToAllLoweringBase<DTensorAllToAllLowering> {
+  void getDependentDialects(mlir::DialectRegistry& registry) const override {
```

Comment:
Is this required? Looks like most of other passes in this file didn't declare the dependent dialects. (I am not super familiar with the dialects. One of our internal test tools says even if this function is removed the affected tests pass.